### ISSUE TITLE
[Bug]: tooltips / info overlay with details in client menu on Mac OS unreadable

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/desktop#8740]
### ⚠️ Before submitting, please verify the following: ⚠️

- [x] This is a **bug**, not a question or a configuration issue.
- [x] This issue is **not** already reported on Github (I've searched it).
- [x] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

When mousing over a entry in the pop-up of the desktop client, the text is nearly unreadable. This is on dark mode, latest Mac OS and client version. Screenshot below.

<img width="1884" height="2456" alt="Image" src="https://github.com/user-attachments/assets/55567a0d-0b83-4499-9036-7aa197e13dea" />

### Steps to reproduce

mouse over an item in the menu on Mac OS with dark mode on.

### Expected behavior

readable text in the overlay. Maybe we can darken the overlay and use white instead of black text... the white text below it is more readable than the black text right now!

### Which files are affected by this bug

any file or activity

### Operating system

macOS

### Which version of the operating system you are running.

26 beta

### Package

Official macOS 12+ universal pkg

### Nextcloud Server version

32

### Nextcloud Desktop Client version

3.17.1

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.16.1 to 3.16.2)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell

```

**Comment by Rello** [comment]
<img width="491" height="138" alt="Image" src="https://github.com/user-attachments/assets/78d13baf-a1bf-49f7-b425-0bd86fcad550" />
Hi Jos,
with the current v4 RC, I do not have this issue anymore. QT was updated so I guess this solved it. closing for now I would say...

**Comment by nilsding** [comment]
@Rello are you using macOS 26 as well?

**Comment by Rello** [comment]
> [@Rello](https://github.com/Rello) are you using macOS 26 as well?

oh. missed that one. will need to update today or so

**Comment by nilsding** [comment]
I saw this on macOS Sequoia (15) as well in a dev build
